nautobot/docs/configuration/optional-settings.md

```diff
@@ -479,6 +479,9 @@ Environment Variable: `NAUTOBOT_MAINTENANCE_MODE`
 
 Setting this to `True` will display a "maintenance mode" banner at the top of every page. Additionally, Nautobot will no longer update a user's "last active" time upon login. This is to allow new logins when the database is in a read-only state. Recording of login times will resume when maintenance mode is disabled.
 
+!!! note
+    The default [`SESSION_ENGINE`](#session_engine) configuration will store sessions in the database, this obviously will not work when `MAINTENANCE_MODE` is `True` and the database is in a read-only state for maintenance.  Consider setting `SESSION_ENGINE` to `django.contrib.sessions.backends.cache` when enabling `MAINTENANCE_MODE`.
```

Comment:
Any thoughts or experience on the benefits and costs of potentially changing the *default* `SESSION_ENGINE` for Nautobot to `backends.cache` instead of its current default of `backends.db` (as proposed in https://github.com/nautobot/nautobot/issues/506#issuecomment-854799681)?

Also, any update that would be appropriate to the `SESSION_ENGINE` docs in this same file (which mention `backends.file` as an alternative but do not explicitly mention `backends.cache`)?

Reply:
@jathanism and I talked about changing the default and decided we still need this update for existing users.  I don't have any experience switching to the cache backend so I really can't comment on that.